src/Components/ReceiverBankAccount/Index.js

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useState } from "react";
```

Comment:
very minor, but let's call this ReceiverBankAccount.js to be consistent with other components